Userland/Demos/Gradient/Gradient.cpp

```diff
@@ -95,12 +95,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(Core::System::unveil("/res", "r"));
     TRY(Core::System::unveil(nullptr, nullptr));
 
-    auto window = TRY(Desktop::Screensaver::create_window("Screensaver"sv, "app-screensaver"sv));
+    auto window = TRY(Desktop::Screensaver::create_window("Gradient"sv, "app-screensaver"sv));
 
-    auto screensaver_window = TRY(window->set_main_widget<Screensaver>(64, 48, 10000));
-    screensaver_window->set_fill_with_background_color(false);
-    screensaver_window->set_override_cursor(Gfx::StandardCursor::Hidden);
-    screensaver_window->update();
+    auto gradient_window = TRY(window->set_main_widget<Gradient>(64, 48, 10000));
```

Comment:
That's because I was copying the Starfield app

Reply:
I know :)

Reply:
I hope I've corrected both `Gradient.cpp` and `Starfield.cpp` to now use `name_widget` instead of `name_window` - but this is guess work as I don't know C++ (yet)! 😀